pipeline-trigger/src/api/amqp/amqpHandler.ts

```diff
@@ -0,0 +1,81 @@
+// import * as AMQP from 'amqplib'
```

Comment:
Why the content of this file is completely commented out?

Reply:
I forgot to set the Pull Request as draft. 
I just changed it. 
Georg wanted to talk about the changes, so I made a pull request. 
It's not cleaned up yet. 
Thanks for the feedback, I will change it.